### ISSUE TITLE
approximants.py, in the series folder, is it correct

### ISSUE THREAD

**Issue by Roland-Sands** [sympy/sympy#14215]
The docs say that it performs Pade approximation
Pade approximants of a function f are a sequence of rational functions which converge to f
In an example, a finite list of Fibonacci numbers is given, and then approximants is applied to the list to create a list of rational functions
Which function is being approximated
This seems very strange to me
When I have seen Pade approximants being used, it would be of the form `Pade(exp(x), 0)`, where it would produce f_1, f_2, f_3, ... , a sequence of rational functions that approximate exp in a neighbourhood of `0`

**Comment by anutosh491** [comment]
Yeah True, Though I am not too well versed in Pade's approximation but your doubt is valid . Even I am confused regarding the doctsring  and not sure enough what the user's could expect out of this . A user wanting an approximation through the `pade's` method ,would want to pass a` function` and the `degrees` of  the numerator and denominator (m,n maybe) . Conventionally Pade's approximation is done in the neighbourhood of `0` but we could also maybe pass a point for the same .
Hence nevertheless I would like a dedicated method for Pade's approximation anyways( I don't think there's an implementation yet, please correct me if there is) seperate from approximants method.

Also the original issue doubt is valid I guess , do we expect `approximants.py` in the series folder ??? As per what I know Pade's approximations for univariate and multivariate cases for series are implemented on/or rather more closer to rings of formal power series and power series(atleast other CAS like sage, maxima etc and research papers demonstrating the algorithm apply it through rings) . Hence I would say this should be somewhere in the `rs_series .py/rings.py` of the `Polys` module

An example given in sage goes as follows
```
sage: z = PowerSeriesRing(QQ, 'z').gen()
sage: exp(z).pade(4, 0)
1/24*z^4 + 1/6*z^3 + 1/2*z^2 + z + 1
```
 For reference https://en.wikipedia.org/wiki/Pad%C3%A9_approximant .
What do you'll think @smichr  @jksuom @0sidharth !
